lib/translator/common.h

```diff
@@ -23,6 +28,34 @@ void DebugLog(const char* format, ...);
 
 void SetDebugCallback(void (*callback)(const char*));
 
+void* AllocPage(uint8_t count);
```

Comment:
Why `uint8_t`? Is that the maximum SCSI can possibly require? Maybe worth a comment explaining this.